.local/matrix-runner.sh

```
@@ -0,0 +1,208 @@
+    return
+  fi
+  # Wait for k6 to start running (or api ready)
+  for i in $(seq 1 150); do
+    kubectl -n redis logs "$k6pod" -c k6 --tail=20 2>/dev/null | grep -qE "running \(|api reachable" && break
+    sleep 1
+  done
+
+  sleep 60  # warmup
+
+  local t0=$(date -u +%s)
+  local rev0=$(ch_q "SELECT count() FROM default.redis_events")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)